### DISCUSSION TITLE
How set DTMF Payload Type

### DISCUSSION THREAD

**Discussion by donacarr** [Freetz-NG/freetz-ng#642] [Allgemein]
Hi there,

do you know if it is possibile to change same parameter in `/var/flash/voip.cfg ` or other place to force the DTMF Payload Type ?
I have an issue when calling many IVR that do not recognize the DTMF tone sent via RTP.
I guess because in the response to the INVITE the payload type is 96 instead of 101 like in the INVITE 

![image](https://user-images.githubusercontent.com/13415391/201296737-13cd9849-b55d-40ce-907c-3b71dd046245.png)

Thank you in advance!

**Comment by fda77** [comment]
Sry, no idea. If you find a way, it is possible with next FOS version it does not work anymore. 
Like Hybird, wihch is not usable anymore with AVM devices as the DSCP tags are messed.
Maybe you better use a reliable software, i switched to asterisk
